packages/cfd/src/Containers/cfd-personal-details-modal.tsx

```diff
@@ -127,10 +128,11 @@ const CFDPersonalDetailsModal = ({
             </div>
             <div className='cfd-personal-details-modal__body'>
                 <CFDPersonalDetailsForm
-                    changeable_fields={getChangeableFields()}
                     form_error={form_error}
                     has_previous_button
                     index={0}
+                    context={context}
+                    // changeable_fields={getChangeableFields()}
```

Comment:
Why did you comment this?

Reply:
I was causing errors and it's bug fix required a lot of time at that time. I have commented it to fix other issues and then get back to it. Will fix it beside compare accounts flow but I need this PR to get merged first so that I can work on the compare accounts.